### DISCUSSION TITLE
[Announcement] We're finally getting the video that we wanted!

### DISCUSSION THREAD

**Discussion by pinuke** [brianpetro/obsidian-smart-connections#98] [Showcase]
@brianpetro Exciting news! The OpenTTRPG Community is helping us out and getting us the video that we wanted with #54!

![image](https://user-images.githubusercontent.com/12621101/227765758-6b2dab10-2a2a-4df2-b8db-c1842b73cc65.png)
![image](https://user-images.githubusercontent.com/12621101/227765779-765898f3-9379-4c0f-a3b6-e5b960ffabb9.png)

### Also vouched for ya bud! I'm excited for you!

![image](https://user-images.githubusercontent.com/12621101/227765724-456aca1a-7012-4c26-8590-2879d5bb4128.png)
![image](https://user-images.githubusercontent.com/12621101/227765800-691d80e2-8786-411b-9805-3741c4eabe1b.png)

@yorrd @Mearman get ready for some publicity lol!

**Comment by Mearman** [comment]
Publicity?

**Reply by brianpetro** [reply]
That would be awesome, thanks!
